packages/admin-panel/src/VizBuilderApp/api/queries/useReportPreview.js

```diff
@@ -20,6 +21,7 @@ export const useReportPreview = (visualisation, project, location, enabled, setE
       return response.results;
     },
     {
+      ...DEFAULT_REACT_QUERY_OPTIONS,
```

Comment:
A nice improvement would be defining our own `query` helper, which uses `DEFAULT_REACT_QUERY_OPTIONS`